internal/controller/finbackup_controller.go

```diff
@@ -681,8 +683,8 @@ func snapIDPreconditionSatisfied(backup *finv1.FinBackup, otherFinBackups []finv
 			return fmt.Errorf("found FinBackup not yet stored to node or verified: %s/%d", fb.Name, snapID)
 		}
 		smallerIDs++
-		if smallerIDs >= 2 {
-			return errors.New("found incremental FinBackup")
+		if smallerIDs >= maxOlderFinBackups {
```

Comment:
IIUC, the max number of older FinBackup is 1. Thus, `maxOlderFinBackups` should be 1 and the comparison will look like:

```suggestion
		if smallerIDs > maxOlderFinBackups {
```